### PR TITLE
Remove the acceptance tests

### PR DESCRIPTION
**Pull request by rupertbates**
## Why are you doing this?
The acceptance tests are no longer useful for this project as all the functionality they were testing has been removed.

This PR removes all the acceptance test code plus some unused AB testing code

### CODE REVIEW

File: frontend/build.sbt [hunk 1]
```diff
@@ -113,8 +111,4 @@ libraryDependencies ++= frontendDependencies.map {
 
 addCommandAlias("devrun", "run 9100")
 
-libraryDependencies ++= acceptanceTestDependencies
-
 addCommandAlias("fast-test", "testOnly -- -l Acceptance")
```
Comment (johnduffell) [inline frontend/build.sbt:114]: do we still need the exclusion now?

**Comment by paulbrown1982** [comment]
👋

File: frontend/build.sbt [hunk 2]
```diff
@@ -113,8 +111,4 @@ libraryDependencies ++= frontendDependencies.map {
...
-
-addCommandAlias("acceptance-test", "testOnly acceptance.JoinSupporterSpec")
```
Comment (johnduffell) [inline frontend/build.sbt:120]: where did the actual file go from `main`? https://github.com/guardian/membership-frontend/blob/868fcf6617a1bbb946ea98cb3342f3f44b32ffc4/frontend/test/acceptance/JoinSupporterSpec.scala

**Review by johnduffell — APPROVED** [review]
nice work this is something that needed doing! 👏 
just a couple of comments above

File: frontend/build.sbt (hunk 2 above)
Comment (rupertbates) [reply frontend/build.sbt:120]: I think it was deleted quite a while ago. It's certainly not in main now.

**Pull request merged**